doc/providers/form.rst

```diff
@@ -133,10 +138,13 @@ form by adding constraints on the fields::
             'constraints' => new Assert\Email()
         ))
         ->add('billing_plan', ChoiceType::class, array(
-            'choices' => array(1 => 'free', 2 => 'small_business', 3 => 'corporate'),
+            'choices' => array('free' => 1, 'small business' => 2, 'corporate' => 3),
```

Comment:
That's correct. http://symfony.com/doc/current/reference/forms/types/choice.html#choices